spec/classes/foreman_plugin_openscap_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::openscap' do
+  on_supported_os.each do |os, facts|
+    if facts[:osfamily] == 'RedHat'
+      context "on #{os}" do
+        let :facts do
```

Comment:
actually this is needed, oitherwise facts are empty, on_supported_os does not seem to set facts (which makes sense because of various contexts)